src/helpers/curve.ts

```diff
@@ -87,28 +87,22 @@ export function getLiquidityDeltaFromAmountA(
   lowerSqrtPrice: BN, // current sqrt price
   upperSqrtPrice: BN // max sqrt price
 ): BN {
-  const prod = new Decimal(
-    maxAmountA.mul(upperSqrtPrice.mul(lowerSqrtPrice)).toString()
-  );
-  const delta = new Decimal(upperSqrtPrice.sub(lowerSqrtPrice).toString());
+  const product = maxAmountA.mul(lowerSqrtPrice).mul(upperSqrtPrice); // Q128.128
```

Comment:
could you rename `maxAmountA ` -> `amountA `

Reply:
updated